packages/smooth_app/lib/generic_lib/widgets/language_selector.dart

```diff
@@ -50,13 +50,16 @@ class LanguageSelectorSettings extends StatelessWidget {
                     .getLanguageNameInEnglishFromOpenFoodFactsLanguage(b)));
         List<OpenFoodFactsLanguage> filteredList = leftovers;
         await showDialog<void>(
-            context: context,
-            builder: (BuildContext context) {
-              return StatefulBuilder(
-                builder: (BuildContext context,
-                    void Function(VoidCallback fn) setState) {
-                  return SmoothAlertDialog(
-                    body: Column(
+          context: context,
+          builder: (BuildContext context) {
+            return StatefulBuilder(
+              builder: (BuildContext context,
+                  void Function(VoidCallback fn) setState) {
+                return SmoothAlertDialog(
+                  body: SizedBox(
+                    height: MediaQuery.of(context).size.height / 2,
```

Comment:
That could be very little on small devices

Reply:
Thanks @M123-dev  for the review 😄 
It looks like it should work fine,
If you say, I could make it full

https://user-images.githubusercontent.com/57723319/191066788-52c6c31c-ce34-4a32-9215-0aba7db5ed93.mp4

About refactoring , if i could do that in a later PR, just did a fix for now, would require some more time for that, running with my university exams.

Reply:
Sounds good to me @AshAman999

Reply:
So maybe we merge it now 
@M123-dev ⁉️

Reply:
👍🏼